### ISSUE TITLE
Chap 9 items for Macartan

### ISSUE THREAD

**Issue by ajacobs71** [macartan/integrated_inferences#100]
- [ ] What is $\lambda_\theta$? Do we need this?

- [ ] And what does “as determined by $\lambda_\theta$” mean? “Determined” in what sense?

- [ ] Consistency in whether we use mathbf for K or other vectors in this chapter?

- [ ] Can you rephrase this more clearly, please: “If we think of $\lambda$ parameters as representing population shares in different populations, we may imagine these as coming from a common distribution parameterized by $\alpha$ for which we have a prior distribution, such as an inverse Gamma prior. Otherwise we might think of $\alpha$ as being a fixed.”

- [ ] In the ambig and param matrices, might we want to use our usual theta notation rather than CQ notation for causal types? And lambdas for the table with the snippet from the posterior? Using CQ notation will need extra translation in the text and there;s enough going on as it is. If we change to theta’s we will need to do some searching and replacing to catch sentences like this: “This distribution can be read directly from column 9 ($Y01$) of Table  \@ref(tab:posteriortable).”

- [ ] Can you clarify the highlighted part of this: “In other words, the likelihood of observing a particular data pattern given $\lambda$ is given by the corresponding value of the multinomial distribution given the data probabilities.” Maybe just one more sentence spelling out that that phrase means.

- [ ] Can you rephrase to make clearer what the highlighted phrase here is meant to evoke:  “Still more complex queries may require keeping some nodes constant while varying others, yet all of these can be calculated as summaries of the combinations of columns of the posterior distribution, following the rules described in Chapter \@ref(questions).”

- [ ] Fig 9.4: adding a row for lots of cases – and text fixed to reflect this.

**Comment by macartan** [comment]
I have done everything except the ambig / param matrices; this is not too easy as we have difficulties with tex in column headers 
Maybe revisit in typesetting
